code/style.css

```diff
@@ -1,3 +1,199 @@
+:root {
+  --bg-color: rgb(13,17,23);
+  --primary-color: rgb(160,168,175);
+}
+
 body {
-  background: #FFECE9;
+  font-family: Helvetica,Arial;
+  margin: 0;
+  padding: 0;
+  background-color: var(--bg-color);
+}
+
+.topnav{
+  width: 100%;
+  height: 100%;
+  background-color: rgb(22,27,34);
+  border: 2px solid transparent;
+  color: var(--primary-color);
+}
+
+h1 {
+  font-size: 40px;
+  text-align: center;
+}
+
+.user-info {
+  margin: 40px 0;
+}
+
+.picture {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.user-picture {
+  width: 250px;
+  border-radius: 50%;
+  margin: 20px 0px;
+}
+
+.user-information {
+  color: var(--primary-color);
+}
+
+.user-information a {
+  text-decoration: none;
+  color: rgb(42, 78, 98);
+  font-weight: bold;
```

Comment:
Here you have font-weight bold and on line 62 font-weight 400, maybe decide which one to use to be consistent? :)